docs/Komponentengalerie/restricted-elements.md

```diff
@@ -0,0 +1,30 @@
+---
+page_id: 55d68d05-6283-4263-b062-0c3321025478
+---
+
+import Restricted from '@tdev-components/documents/Restricted';
+import BrowserWindow from '@tdev-components/BrowserWindow';
+
+# Zugriffsbeschränkte Elemente
+
+Wenn ein Element nur nach einer Freigabe angezeigt werden soll, eignet sich die `<Restricted>`-Komponente - sie versteckt ihren Inhalt, bis er freigegeben wurde.
+
+```md
+<Restricted id="43ba2a38-9612-402f-9daf-91204ff47a71">
+Dieser Link wird nur bei entsprechender Berechtigung angezeigt:
+
+[🔗 https://test.com/nicht-oeffentlicher-link](https://test.com/nicht-oeffentlicher-link)
+</Restricted>
+```
+
+<BrowserWindow>
+<Restricted id="43ba2a38-9612-402f-9daf-91204ff47a71">
+Dieser Link wird nur bei entsprechender Berechtigung angezeigt:
+
+[🔗 https://test.com/nicht-oeffentlicher-link](https://test.com/nicht-oeffentlicher-link)
+</Restricted>
+</BrowserWindow>
+
+:::warning[Inhalt ist nicht geheim]
+Obwohl der zugriffsbeschränkte Inhalt bei fehlender Berechtigung nicht im Browser angezeigt wird, ist er dennoch im Seitenquelltext enthalten. Versierte Nutzer:innen können also trotzdem darauf zugreifen.
```

Comment:
Haben wir source-maps aktiv? Andernfalls ist der Inhalt nur in den minified js files zu finden... das ist nochmals schwieriger zu finden... aber auch mit aktiven source-maps ist es nicht offensichtlich, wo der Inhalt zu finden ist...

Reply:
Guter Punkt! Ich habe das in der Warnung noch etwas "abgeschwächt" und auf der Gallery-Page die Use Cases aus diesem PR ergänzt. Damit sollte relativ klar sein, für welche Elemente man diese Komponenten problemlos verwenden darf, und wo es vielleicht zu riskant wäre.